distribution/osbuild-composer-koji.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=OSBuild Composer Koji
+After=multi-user.target
+Requires=osbuild-composer-koji.socket
+
+[Service]
+Type=simple
+ExecStart=/usr/libexec/osbuild-composer/osbuild-composer-koji
+CacheDirectory=osbuild-composer
+StateDirectory=osbuild-composer
```

Comment:
Does it make sense to share these directories? `fsjobqueue` does not support multiple processes on the same directory.

Reply:
You are so right.

Reply:
Fixed.